src/app/static/src/app/components/load/LoadErrorModal.vue

```diff
@@ -2,15 +2,46 @@
     <div id="load-error-modal">
         <modal :open="hasError">
             <h4 v-translate="'loadError'"></h4>
-            <p>{{ loadError }}</p>
+            <p v-if="showInvalidSteps">
+              <span id="load-error-steps" v-translate="'loadErrorSteps'" />
+              <ul id="load-error-steps-list">
+                <li v-for="step in invalidSteps" :key="step" v-translate="stepTextKey(step)"></li>
+              </ul>
+              <template v-if="lastValidStep >= 1">
+                <span id="load-error-steps-from-valid-action" v-translate="'loadErrorStepsFromValidAction'" />
+                <span id="load-error-last-valid" v-translate="stepTextKey(lastValidStep)" />.
+                <span id="load-error-steps-from-valid-warning" v-translate="'loadErrorStepsFromValidWarning'" />
+              </template>
+              <span v-else id="load-error-steps-all-action" v-translate="'loadErrorStepsAllAction'" />
+            </p>
+            <p v-else id="load-error-error">{{ loadError }}</p>
             <template v-slot:footer>
-                <button type="button"
+              <template v-if="showInvalidSteps">
+                <button id="retry-load"
+                        type="button"
                         class="btn btn-red"
-                        data-dismiss="modal"
-                        aria-label="Close"
-                        @click="clearLoadError"
-                        v-translate="'ok'">
+                        aria-label="Retry"
+                        @click="retryLoad"
+                        v-translate="'retry'">
                 </button>
+                <button id="rollback-load"
+                        type="button"
+                        class="btn btn-red"
+                        aria-label="Rollback"
+                        @click="rollbackInvalidState"
+                        v-translate="'rollback'">
+                </button>
+              </template>
+              <template v-else>
+                  <button id="ok-load-error"
+                          type="button"
+                          class="btn btn-red"
+                          data-dismiss="modal"
+                          aria-label="Close"
+                          @click="clearLoadError"
+                          v-translate="'ok'">
+                  </button>
+              </template>
```

Comment:
What is this button for? I couldn't get into a state where this would show

Reply:
This could be shown if you tried to load a corrupted zip file for example.